addons/addon-base-raas/packages/base-raas-services/lib/schema/update-storage-gateway.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
There should be some fields that are required here, right?

Reply:
For updates, usually only id and rev number are required. (e.g. https://github.com/awslabs/service-workbench-on-aws/blob/master/addons/addon-base-raas/packages/base-raas-services/lib/schema/update-environment-sc.json). In our case, since id and rev were passed in separately, not sure which fields should be required here?

Reply:
I guess for your case `fileShares` is must since that is what we expect to be updated?

Reply:
Talked with Jeet off line and decided to leave it as is, since there might be other use cases in the future that does not update fileShares attribute. Will leave it to be updated later when we know more of the use cases.